### DISCUSSION TITLE
TypeError: Cannot read properties of undefined (reading '700')

### DISCUSSION THREAD

**Discussion by Snr-Mn** [postcss/postcss#1895] [Q&A]
I'm using **NextJS V13** with **"PostCSS": "^8.3.6"** and **"tailwindcss": "^2.2.19"**

My code was running perfectly before suddenly giving me this error:
```
./node_modules/next/dist/build/webpack/loaders/css-loader/src/index.js??ruleSet[1].rules[2].oneOf[7].use[1]!./node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js??ruleSet[1].rules[2].oneOf[7].use[2]!./node_modules/nprogress/nprogress.css`
TypeError: Cannot read properties of undefined (reading '700')
    at Array.reduce (<anonymous>)
```

Here is my package.json:
  ```
  "dependencies": {
    "@tailwindcss/forms": "^0.3.3",
    "@tailwindcss/line-clamp": "^0.2.1",
    "@tailwindcss/typography": "^0.4.1",
    "next": "^12.3.0",
    "nprogress": "^0.2.0",
    "sass": "^1.39.0",
  },
  "devDependencies": {
    "autoprefixer": "^10.4.16",
    "postcss": "^8.4.31",
    "prettier": "^2.3.2",
    "tailwindcss": "^3.3.3"
  }

```

I've tried cleaning my cache, deleting my Node_Module and installing it again, upgrading and downgrading my packages, but nothing seems to fix the error. 

I've also looked into a similar issue, but it did not help my case.
`https://github.com/tailwindlabs/tailwindcss/issues/6398`

**Comment by ai** [comment]
Sorry, the stack trace is not pointing to the PostCSS core and I can't help with environment debug
